lab2/mathematicalAnalysis.tex

```diff
@@ -28,5 +28,29 @@
 	
 	It is clear by inspection that $a_0$ evaluates to 0. Similarly, the square wave is an odd function, and it can be shown that $a_n = 0$ for all $n$.
 	
+	$$b_n = \frac{2}{T} \int_{0}^{T} sin(\frac{n \pi t}{T}) dt$$
+	$$f = \frac{1}{T}$$
+	$$b_n = \frac{1}{T} \int_{0}^{T} sin(n \pi f t) dt$$
+	$$b_n = \frac{2}{T} \Big(\frac{-1}{n \pi f}\Big) [cos(n \pi f t)] \Big|_0^T$$
+	$$b_n = \frac{-2}{n \pi} [cos(n \pi) - cos(0)]$$
+	
+	$$\text{For all even $n$, } b_n = 0$$
+	$$\text{For odd $n$ use substitution $n = 2k-1$}$$
+	$$b_k = \frac{-2}{(2k-1) \pi} \Big(cos((2k-1) \pi) - 1 \Big)$$
+	$$cos((2k-1) \pi) = -1 \text{ for all $k$}$$
+	$$b_k = \frac{-2}{(2k-1) \pi} \Big(-2 \Big)$$
+	$$b_k = \frac{4}{(2k-1) \pi}$$
+	\\
+	Substituting $\omega_0 = \frac{2 \pi}{T}$ and $f = \frac{1}{T}$ into the sine terms of the earlier definition of the Fourier Series we get,
+	
+	$$f(t) = \sum_{k=1}^{\infty} \frac{4}{(2k-1) \pi} sin\Big(\frac{2 \pi (2k-1)}{T}t \Big)$$
+	
+	
+	For $k=1$   $1.2732 sin(2 \pi f)$
```

Comment:
What was the intention of this paragraph? Also was this supposed to be a table? If not the formatting is a bit off on my machine (the spacing between the constants and the sines are really crammed together.